.github/workflows/ci.yml

```diff
@@ -120,5 +121,9 @@ jobs:
           tar xf targets.tar
           rm targets.tar
 
-      - name: Publish project
-        run: sbt --client '++${{ matrix.scala }}; +publish'
\ No newline at end of file
+      - env:
```

Comment:
```suggestion
      - name: Publish project
        env:
```

Reply:
This is generated file, I wouldn't edit it by hand :)